content/applications/finance/accounting/fiscal_localizations/localizations/germany.rst

```diff
@@ -31,7 +31,134 @@ Export from Odoo to Datev
 It is possible to export your accounting entries from Odoo to Datev. To be able to use this
 feature, the german accounting localization needs to be installed on your Odoo Enterprise database.
 Then you can go in :menuselection:`Accounting --> Reporting --> General Ledger` then click on the
-**Export Datev (csv)** button. 
+**Export Datev (csv)** button.
+
+.. _germany/pos:
+
+Point of Sale in Germany: Technical Security System
+===================================================
+
+Since the new regulations "The Act on Protection against Manipulation of Digital Records"
+(Kassensicherungsverordnung), a **Technical Security System** (also called **TSS** or **TSE**) needs
+to be used for each :doc:`point of sale </applications/sales/point_of_sale>`.
+
+Odoo offers a service that is compliant with the help of `fiskaly <https://fiskaly.com>`_, a
+*cloud-based solution*.
+
+.. important::
+   Since this solution is cloud-based, a working internet connection is required.
+
+.. note::
+   The only VAT rates allowed are given by fiskaly. You can check these rates by consulting:
+   `fiskaly DSFinV-K API: VAT Definition
+   <https://developer.fiskaly.com/api/dsfinvk/v0/#tag/VAT-Definition>`_.
+
+Configuration
+-------------
+
+Modules installation
+~~~~~~~~~~~~~~~~~~~~
+
+#. If your database was created before June 2021, :ref:`upgrade <general/upgrade>` your **Point of
+   Sale** app (`point_of_sale`) and the **Restaurant** module (`pos_restaurant`).
+#. :ref:`Install <general/install>` the **Germany - Certification for Point of Sale**
+   (`l10n_de_pos_cert`)and **Germany - Certification for Point of Sale of type restaurant**
```

Comment:
```suggestion
   (`l10n_de_pos_cert`) and **Germany - Certification for Point of Sale of type restaurant**
```
Small typo

Reply:
thanks @trj-odoo .
and is it okay to have written "before June 2021"?
I wanted to ask you if that was correct since I added it out of the blue.

Reply:
Yes, it's ok for me

Reply:
Thanks